apps/user-office-frontend/src/components/review/ProposalTableReviewer.tsx

```diff
@@ -213,19 +210,25 @@ const ProposalTableReviewer: React.FC<{ confirm: WithConfirmType }> = ({
     </>
   );
 
-  const updateView = () => {
-    if (currentAssignment) {
-      const currentReview = (currentAssignment as SepAssignment).review;
+  const updateView = async (reviewId?: number | null) => {
+    if (!reviewId) {
+      return;
+    }
+
+    const { review: freshReview } = await api().getReview({
+      reviewId: reviewId,
+    });
 
+    if (freshReview) {
       const userDataUpdated = {
         ...userData,
         reviews: userData?.reviews.map((review) => {
-          if (review.id === currentReview?.id) {
+          if (review.id === freshReview?.id) {
```

Comment:
It seems odd to me that this line is using a `?.` null check on `freshReview` when we're already inside an `if (freshReview)` block. Would `freshReview.id` be enough here?